client/src/main/java/nl/altindag/client/ClientConfig.java

```diff
@@ -212,6 +218,41 @@ public Client jerseyClient(@Autowired(required = false) SSLFactory sslFactory) {
         }
     }
 
+    /**
+     * JAX-RS configuration should be identical to {@link #jerseyClient(SSLFactory)} once CXF update to version 3.5.0
+     * But this function is still necessary to create CXF version of client that does not depend on Java SPI (/META-INF/services/javax.ws.rs.client.ClientBuilder)  
+     */
+    @Bean
+    @Qualifier("cxf")
+    public javax.ws.rs.client.Client cxfJaxRsClient(@Autowired(required = false) SSLFactory sslFactory) {
+    	// One can just use ClientBuilder.newBuilder(), Explicit use here is due to multiple JAX-RS implementations in classpath
+    	javax.ws.rs.client.ClientBuilder clientBuilder = new org.apache.cxf.jaxrs.client.spec.ClientBuilderImpl();
+    	 if (nonNull(sslFactory)) {
+    		 clientBuilder
+    		 	.sslContext(sslFactory.getSslContext())
+                .hostnameVerifier(sslFactory.getHostnameVerifier());
```

Comment:
The indentation looks a bit off here... 😄

Reply:
Sorry .  it is tabs.   
What do you have here for this project ?

![image](https://user-images.githubusercontent.com/6963441/103589655-e605fb00-4eb9-11eb-88b7-accb88e9a0e7.png)

Reply:
I use 4 spaces as tab and I am not using Eclipse so not quite sure if it would be helpful to share my formatter...

The unit test `createCxfJaxRsClientWithoutSecurity` contains 5 spaces as indentation, `createCxfJaxRsClientWithSecurity` also 5 spaces. `createCxfWebClientWithoutSecurity` uses a mixed of 4 and 5 spaces etc. I would recommend to use 4 spaces to align with the remaining code base..

Reply:
Reformatted the code that had been changed with IntelliJ default settings  -  hope it is good now.